n3fit/src/n3fit/model_gen.py

```diff
@@ -37,6 +40,7 @@ class ObservableWrapper:
 
     name: str
     observables: list
+    trvl_mask_layers: list
```

Comment:
Maybe we can have this be optional = None, and if it is None then it doesn't get applied.

Because (I think, maybe I'm wrong!) that we should never be in a situation in which some of the masks exist and some are `None`, that way you can avoid the list of `None`.

Reply:
Fixed in rev. bc568761540fa4a96e91963ad314041fa81dc782. I did not insert the default value yet though